tests/tests_ntp_provider6.yml

```diff
@@ -0,0 +1,73 @@
+
+- name: Configure NTP with OS release non-default provider and then change it to the default provider
+  hosts: all
+  vars:
+    is_el6: "ansible_distribution in ['RedHat', 'CentOS'] and ansible_distribution_version is version('7.0', '<')"
```

Comment:
Would it make sense to rename this variable to something less OS specific? Maybe `is_ntp_default`?

Reply:
Good idea, it will make it more obvious why there is this test. @vcrhonek it would be great if you could change this.

Reply:
I agree, updated.